products/views.py

```diff
@@ -104,10 +107,59 @@ def get(self, request, product_group_id):
                         "related_product_name"      : relate_product.related_product_group.name,
                         "related_product_price"     : relate_product.related_product_group.price,
                         "related_product_thumbnail" : relate_product.related_product_group.thumbnail
-                    } for relate_product in product_group.relate.all().select_related("related_product_group")]
+                    } for relate_product in product_group.relate.all().select_related("related_product_group")],
```

Comment:
```suggestion
                    } for relate_product in product_group.relate.select_related("related_product_group")],
```
꼭 고쳐야 하는 건 아닌데, `all()` 없이도 동작 합니다 :)